examples/src/guest-book-borsh.ts

```diff
@@ -0,0 +1,82 @@
+import { NearBindgen, near, call, view, Vector } from 'near-sdk-js';
+import * as borsh from 'borsh';
+
+@NearBindgen({
+  serializer(value) {
+    return borsh.serialize(schema, value);
+  },
+  deserializer(value) {
+    return borsh.deserialize(schema, value);
+  },
+})
+class GuestBook {
+  messages: Vector<PostedMessage> = new Vector<PostedMessage>('v-uid');
+
+  @call({ payableFunction: true })
+  // Public - Adds a new message.
+  add_message({ text }: { text: string }) {
+    // If the user attaches more than 0.1N the message is premium
+    const premium = near.attachedDeposit() >= BigInt(POINT_ONE);
+    const sender = near.predecessorAccountId();
+
+    const message: PostedMessage = { premium, sender, text };
+    this.messages.push(message, {
+      serializer: (value) => borsh.serialize(MessageSchema, value),
```

Comment:
This is a manual serialization, can we make use of above
```
@NearBindgen({
  serializer(value) {
    return borsh.serialize(schema, value);
  },
```
to auto serialize?

Reply:
didn't catch that, could you clarify?